### PR TITLE
Convert Java-like code to Kotlin-like code

### PR DESCRIPTION
**Pull request by ChachyDev**

### CODE REVIEW

File: src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt [hunk 1]
```diff
@@ -219,8 +218,8 @@ class ChatListener {
                 val lines = (chatGui as AccessorGuiNewChat).chatLines
                 val drawnLines = (chatGui as AccessorGuiNewChat).drawnChatLines
                 var i = 0
-                delay(Duration.microseconds(100))
-                while (!StringUtils.stripControlCodes(lines[i].chatComponent.unformattedText)
+                delay(100)
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt]: Isn't delay defaulted to milliseconds instead of microseconds

**Review by My-Name-Is-Jeff — APPROVED** [review]

**Comment by Sychic** [comment]
Why are 90% of the changes just adding a 1 to the variable names and changing imports

File: src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt (hunk 1 above)
Comment (Sychic) [reply src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt]: yeah I wanted it to be slightly less than a millisecond so it'd clear the lines faster

**Review by Sychic — APPROVED** [review]

File: src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt (hunk 1 above)
Comment (ChachyDev) [reply src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt]: ah yeah, woops i'll fix that now. should be resolved in the latest push now

**Pull request merged**